src/Phing/Task/Ext/Analyzer/Sonar/SonarTask.php

```diff
@@ -275,7 +275,7 @@ private function validateExecutable()
 
         $isOk = false;
         foreach ($output as $line) {
-            if (preg_match('/Sonar(Qube )?Scanner \d+\\.\d+/', $line) === 1) {
+            if (preg_match('/Sonar(Qube )?Scanner CLI \d+\\.\d+/', $line) === 1) {
```

Comment:
Do we need to support both old & new versions, or is supporting only the latest / new version of this tool correct?
```suggestion
            if (preg_match('/Sonar(Qube )?Scanner( CLI)? \d+\\.\d+/', $line) === 1) {
```

Reply:
I have no idea when the tool banner was changed, and whether the version with the previous banner is still widely used.  The safe option would be to support old & new versions.